src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProvider.cs

```diff
@@ -178,5 +185,25 @@ internal static OrchestrationInstanceStatusQueryCondition ConvertWebjobsDurableC
                 FetchInput = condition.ShowInput,
             };
         }
+
+#if !FUNCTIONS_V1
+        /// <inheritdoc/>
+        public override bool TryGetScaleMonitor(
+            string functionId,
+            string functionName,
+            string hubName,
+            string storageConnectionString,
+            DurableTaskExtension extension,
```

Comment:
Can we remove `DurableTaskExtension extension` from the list of parameters? Seems like it's not actually being used, and I won't think we necessarily want to give 3rd parties access to this.

Reply:
I kept it in because it looks like the corresponding PR in Netherite that implements `TryGetScaleMonitor()` uses `extension`.

PR - [Support autoscaling on elastic premium plans #24](https://github.com/microsoft/durabletask-netherite/pull/24/)

[TryGetScaleMonitor implementation in that PR](https://github.com/microsoft/durabletask-netherite/pull/24/files#diff-f47141027cfddf5df3bff1836fc7931ca6c1622f56eb3f60cf192b4d2850461dR105)

Reply:
It looks like it was just using it for logging purposes, so they can move to using a logger they create themselves.